manifests/init.pp

```diff
@@ -110,6 +113,7 @@
   String $django_secret_key = extlib::cache_data('pulpcore_cache_data', 'secret_key', extlib::random_password(32)),
   Integer[0] $redis_db = 8,
   Stdlib::Fqdn $servername = $facts['fqdn'],
+  Stdlib::Absolutepath $allowed_import_path = '/var/lib/pulp/sync_imports',
   Optional[String] $remote_user_environ_name = undef,
```

Comment:
@ekohl: I was trying to look it up but how to do I define this to be an array of absolute paths?

Reply:
Looks like I can do this https://github.com/theforeman/puppet-pulpcore/blob/c27d67a6c9705b99c942729dd539698e734f6bd3/manifests/admin.pp#L28

Reply:
Exactly. I love it when people can answer their own questions :)